src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
```

Comment:
Missing header comments for the class. Do remember to add them for `Block`, `PostalCode`, `Street` and `Unit` classes.